central/reports/handler/download.go

```diff
@@ -0,0 +1,125 @@
+package handler
```

Comment:
I would recommend the report APIs to be kept together by moving this file under `reports/service/v2/` pkg. The issue with the existing pattern of `/handler/` in our codebase is that locating all related APIs becomes harder.